core/src/main/java/nl/nn/adapterframework/doc/model/FrankDocModel.java

```diff
@@ -8,42 +9,145 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.StringTokenizer;
 import java.util.stream.Collectors;
 
+import org.apache.commons.lang3.StringUtils;
 import org.apache.logging.log4j.Logger;
 import org.springframework.core.annotation.AnnotationUtils;
+import org.xml.sax.SAXException;
 
 import lombok.Getter;
 import lombok.Setter;
+import nl.nn.adapterframework.configuration.digester.DigesterRule;
+import nl.nn.adapterframework.configuration.digester.DigesterRulesHandler;
+import nl.nn.adapterframework.core.Resource;
 import nl.nn.adapterframework.doc.IbisDoc;
 import nl.nn.adapterframework.doc.IbisDocRef;
+import nl.nn.adapterframework.doc.Utils;
+import nl.nn.adapterframework.doc.objects.SpringBean;
 import nl.nn.adapterframework.util.LogUtil;
+import nl.nn.adapterframework.util.XmlUtils;
 
 public class FrankDocModel {
-	private static final String JAVA_STRING = "java.lang.String";
-
 	private static Logger log = LogUtil.getLogger(FrankDocModel.class);
-
+	private static final String DIGESTER_RULES = "digester-rules.xml";
+	
+	private @Getter Map<String, ConfigChildSetterDescriptor> configChildDescriptors;
 	private @Getter List<FrankDocGroup> groups;
```

Comment:
Ik denk dat groups een Map<Class, List<ElementType>> moet zijn, waarbij de key de interface van het child element is, of 'null' voor de elementTypes die niet een interface representeren, maar een class. De group met class null wordt dan de group 'overig'

Reply:
Gedaan.

Reply:
Maar ik heb niet `Class` als key gebruikt. Die mogen we niet in het model stoppen omdat het dan niet meer voor compile-time Frank!Doc gebruikt kan worden.